js/components/portal-dashboard/all-responses-popup/popup-class-nav.tsx

```diff
@@ -92,10 +83,8 @@ export class PopupClassNav extends React.PureComponent<IProps, IState>{
   }
 
   private renderSpotlightToggle() {
-    let spotLightContainerClasses: string = css.spotlightContainer;
-    if (this.state.isSpotlightOn && this.state.studentSelected) {
-      spotLightContainerClasses += " " + css.spotlightOn;
-    }
+    const spotLightContainerClasses = `${css.spotlightContainer}` + (this.props.isSpotlightOn ? ` ${css.spotlightOn}` : "");
+
     return (
       <div className={`${css.spotlightToggle}`} onClick={this.handleSpotlightClick} data-cy="spotlight-toggle">
```

Comment:
With function currying becomes simply: 
`<div className={`${css.spotlightToggle}`} onClick={this.props.onShowDialog} data-cy="spotlight-toggle">`

Reply:
And cleaning up the class name:
```
<div className={css.spotlightToggle} onClick={this.props.onShowDialog} data-cy="spotlight-toggle">
```